TombEditor/Controls/Panel3D/Panel3DText.cs

```diff
@@ -53,7 +53,7 @@ private static string GetObjectRotationString(Room room, PositionBasedObjectInst
         private static string GetObjectPositionString(Room room, PositionBasedObjectInstance instance)
         {
             // Get the distance between point and floor in units
-            int height = (int)(instance.Position.Y - GetFloorHeight(room, instance.Position)) / (int)Level.HeightUnit;
+            float height = (instance.Position.Y - GetFloorHeight(room, instance.Position)) / Level.FullClickHeight;
```

Comment:
Why this variable was converted to float, if in all other parts coordinates are seemingly converted to int?

Reply:
To display that the object is 0.25 clicks about the ground, otherwise it would only show 0.